cmd/binapi-generator/main.go

```diff
@@ -18,53 +18,64 @@ import (
 	"flag"
 	"fmt"
 	"os"
-	"path/filepath"
-	"strings"
-	"unicode"
 
 	"github.com/sirupsen/logrus"
 
 	"go.fd.io/govpp/binapigen"
 	"go.fd.io/govpp/binapigen/vppapi"
 	"go.fd.io/govpp/version"
+	_ "go.fd.io/govpp/binapigen/plugins"
 )
 
 func init() {
 	flag.Usage = func() {
-		fmt.Fprintf(os.Stderr, "USAGE\n")
-		fmt.Fprintf(os.Stderr, "  Parse API_FILES and generate Go bindings\n")
-		fmt.Fprintf(os.Stderr, "  Provide API_FILES by file name, or with full path including extension.\n")
-		fmt.Fprintf(os.Stderr, "  %s [OPTION] API_FILES\n\n", os.Args[0])
+		fmt.Fprintf(os.Stderr, "GoVPP api bindings generation CLI\n")
+		fmt.Fprintf(os.Stderr, "---------------------------------\n\n")
+		fmt.Fprintf(os.Stderr, "This CLI generates VPP api Go bindings based on .api files or a VPP repository\n\n")
```

Comment:
CLI is just something user interacts with, but generation is done by _the tool_. So..  _This tool generates.._

And for the latter part, we actually support only `.api.json` files at the moment.